recipe/meta.yaml

```diff
@@ -13,18 +13,27 @@ source:
 
 build:
   number: 0
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
So we have been moving to `pip` particularly for `noarch: python`. Not sure if there are concerns making this switch.

ref: https://conda-forge.org/docs/meta.html#use-pip